src/Cli/dotnet/Commands/Run/CommonRunHelpers.cs

```diff
@@ -17,7 +17,6 @@ internal static class CommonRunHelpers
     public static Dictionary<string, string> GetGlobalPropertiesFromArgs(MSBuildArgs msbuildArgs)
     {
         var globalProperties = msbuildArgs.GlobalProperties?.ToDictionary() ?? new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-        globalProperties[Constants.EnableDefaultItems] = "false"; // Disable default item globbing to improve performance
```

Comment:
Consider adding some tests that would reproduce the issue which this is fixing.

Reply:
I will probably add a test upstream in dotnet/android, I don't know how you'd reproduce the problem without the Android workload:
* https://github.com/dotnet/android/issues/10500